### PR TITLE
EM-2066: Fix for zoom issue on outline for PDFs

### PR DESCRIPTION
**Pull request by tomxelliott**
- tests updated
	- source code for pdf js referred to in order to make sense of pdf outline navigation logic

**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: projects/media-viewer/src/lib/viewers/pdf-viewer/pdf-js/pdf-js-wrapper.ts [hunk 1]
```diff
@@ -91,18 +94,32 @@ export class PdfJsWrapper {
     this.pdfViewer.eventBus.dispatch('findbarclose');
   }
 
+  public navigateTo(destination: any) {
+    if (destination instanceof Object) {
```
Comment (linusnorton) [inline projects/media-viewer/src/lib/viewers/pdf-viewer/pdf-js/pdf-js-wrapper.ts]: this seems a bit weird, what else comes in?

Comment (tomxelliott) [reply projects/media-viewer/src/lib/viewers/pdf-viewer/pdf-js/pdf-js-wrapper.ts]: At certain times the destination is only a number that represents the location to navigate to, at others it is an object. I encountered this when testing on several different document types.

Comment (linusnorton) [reply projects/media-viewer/src/lib/viewers/pdf-viewer/pdf-js/pdf-js-wrapper.ts]: maybe update the type info to reflect that, something like `Object | number` or `object | number` - they're different, I can't remember which is correct

File: projects/media-viewer/src/lib/toolbar/side-bar/outline-view/outline-item/outline-item.component.ts [hunk 2]
```diff
@@ -26,7 +26,7 @@ export class OutlineItemComponent implements OnInit {
 
   navigateLink() {
     if (this.outline.dest) {
-      this.linkService.navigateTo(this.outline.dest);
+      this.pdfJsWrapperFactory.navigateTo(this.outline.dest);
```
Comment (velimd) [inline projects/media-viewer/src/lib/toolbar/side-bar/outline-view/outline-item/outline-item.component.ts]: if youre using pdfjswrapperfactory, you can get rid of the link service variable and initialise in ngoninit

File: projects/media-viewer/src/lib/viewers/pdf-viewer/pdf-js/pdf-js-wrapper.ts (hunk 1 above)
Comment (tomxelliott) [reply projects/media-viewer/src/lib/viewers/pdf-viewer/pdf-js/pdf-js-wrapper.ts]: thanks for the comment. this has been updated.

File: projects/media-viewer/src/lib/toolbar/side-bar/outline-view/outline-item/outline-item.component.ts (hunk 2 above)
Comment (tomxelliott) [reply projects/media-viewer/src/lib/toolbar/side-bar/outline-view/outline-item/outline-item.component.ts]: thanks for the comment, this has been updated

**Pull request merged**